graph/db/models/stats.go

```diff
@@ -35,3 +35,13 @@ type NetworkStats struct {
 	// NumZombies is the number of zombie channels in the graph.
 	NumZombies uint64
 }
+
+// BetweennessCentrality represents the betweenness centrality of a node in the
+// graph.
+type BetweennessCentrality struct {
```

Comment:
Same comment for centrality. Feels more like an application level feature and less of a source feature.